### PR TITLE
Rename AuthorizationContext

### PR DESCRIPTION
**Pull request by BrennanConroy**
#806 
@Tratcher @Eilon

### CODE REVIEW

File: src/Microsoft.AspNetCore.Authorization/AuthorizationContext.cs [hunk 1]
```diff
@@ -11,13 +11,13 @@ namespace Microsoft.AspNetCore.Authorization
     /// <summary>
     /// Contains authorization information used by <see cref="IAuthorizationHandler"/>.
     /// </summary>
-    public class AuthorizationContext
+    public class AuthorizationHandlerContext
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Authorization/AuthorizationContext.cs]: Rename the file as well.

**Comment by Eilon** [comment]
I did this search: https://github.com/search?utf8=%E2%9C%93&q=user%3Aaspnet+AuthorizationContext+-repo%3Aaspnet%2Fdocs+-repo%3Aaspnet%2Fazurecache+-repo%3Aaspnet%2Fsecurity&type=Code&ref=searchresults

And I *think* that this needs to be updated as well:
* https://github.com/aspnet/Mvc/blob/a78f77afde003c4a3fcf5dd7b6dc13dd9c85f825/test/WebSites/FiltersWebSite/ManagerHandler.cs#L14

I couldn't find any other hits.

**Comment by Eilon** [comment]
Otherwise, :shipit:

**Comment by dougbu** [comment]
@BrennanConroy have you done the MVC reaction @Eilon mentioned? MVC isn't currently building...

**Comment by BrennanConroy** [comment]
https://github.com/aspnet/Mvc/commits/dev

**Comment by dougbu** [comment]
Thanx

**Pull request merged**